<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xToolsCrcCalculatorToolUi</class>
 <widget class="QWidget" name="xToolsCrcCalculatorToolUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Start index</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="spinBoxStartIndex"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>End index</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBoxEndIndex"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Algorithm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="xToolsCrcAlgorithmComboBox" name="comboBoxAlgorithm"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBoxBigEndian">
     <property name="text">
      <string>BigEndian</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xToolsCrcAlgorithmComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">xToolsCrcAlgorithmComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
